### ISSUE TITLE
Have loom open a new non-interactive R session to render reports

### ISSUE THREAD

**Issue by wfulp** [thebioengineer/loom#3]
It would be nice to get loom to run the render in a new non-interactive R session, like the knit button dose. Here is the current script I run to get all formats to run:
```
Rscript -e "library(rmarkdown); render('my_report.Rmd', output_format = 'all')"
```

**Comment by thebioengineer** [comment]
some guy complained about not having the output go to his console...working on an update

**Comment by thebioengineer** [comment]
resolved with #4
